extensions/amp-ad/0.1/amp-ad-api-handler.js

```diff
@@ -110,11 +110,15 @@ export class AmpAdApiHandler {
       this.iframe_.style.visibility = 'hidden';
     }
     listenForOnce(this.iframe_, 'render-start', () => {
+      this.renderStart_ = true;
       if (!this.iframe_) {
         return;
       }
       this.iframe_.style.visibility = '';
       this.sendEmbedInfo_(this.baseInstance_.isInViewport());
+      if (this.baseInstance_.renderStartResolve_) {
+        this.baseInstance_.renderStartResolve_();
```

Comment:
set `this.baseInstance_.renderStartResolve_` to null after calling it (since Promise should not be resolved more than once, there is no need to keep this allocation)